src/SDK/Metrics/MetricExporterFactoryInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Metrics;
+
+interface MetricExporterFactoryInterface
+{
+    public function fromEnvironment(): MetricExporterInterface;
```

Comment:
`fromEnvironment` might not be the best name, given recent Environment->Configuration changes. Maybe just `::create()` ?